requirements.in

```diff
@@ -20,7 +20,7 @@ factory-boy
 fakeredis
 freezegun
 https://github.com/codecov/opentelem-python/archive/refs/tags/v0.0.4a1.tar.gz#egg=codecovopentelem
-https://github.com/codecov/shared/archive/ebafd805394f22b4ab9e1f6e9a864ab28396dfe4.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/ce68890c789aa1bc187438e71db13de2d0110d1c.tar.gz#egg=shared
```

Comment:
This might need to get updated to a newer version

Reply:
I don’t think we need the update at all. The dependency is the other way around. https://github.com/codecov/codecov-api/pull/1006 is removing an API usage before https://github.com/codecov/shared/pull/415 can remove that code.